Support aggregations (#202)
logstash-plugins/logstash-input-elasticsearch: CHANGELOG.md

## 4.20.0

>   - Added `response_type` configuration option to allow aggregations to be executed

Author: ```suggestion
  - Added `response_type` configuration option to allow processing result of aggregations
```

>   - Added `response_type` configuration option to allow processing result of aggregations

Author: ```suggestion
  - Added `response_type` configuration option to allow processing result of aggregations [#202](https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/202)
```
